vclib-openid/src/commonTest/kotlin/at/asitplus/wallet/lib/oidc/EudiwPidCredentialScheme.kt

```diff
@@ -7,5 +7,79 @@ object EudiwPidCredentialScheme : ConstantIndex.CredentialScheme {
     override val vcType: String = "EudiwPid1"
     override val isoNamespace: String = "eu.europa.ec.eudiw.pid.1"
     override val isoDocType: String = "eu.europa.ec.eudiw.pid.1"
-    override val claimNames: Collection<String> = listOf("family_name")
+    override val claimNames: Collection<String> = listOf(
```

Comment:
why not use the artifact of https://github.com/a-sit-plus/eu-pid-credential?

Reply:
Should this really be a dependency of the vclib?
What if the EUDIW PID credential changes in the future? This may break the tests, even though the tested functionality still remains correct?
Maybe it would be better to just rename it to TestCredential or something like that?

Reply:
Then why not rename it to "TestCredential", or if it tests something EU-PID-specific, move the test to the eu-pid repository? But imho depending on version 1.0.0 of that EU PID artifact isn't that bad, because future updates to that credential should remain backwards-compatible, or be released as 2.0.0, so it won't be picked up by vclib.

Reply:
Has been fixed somewhere along the way, not sure exactly where